kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-common/src/main/java/org/kie/server/remote/rest/common/util/RestUtils.java

```diff
@@ -264,4 +265,19 @@ public static String errorMessage(Throwable e, String defaultMessage) {
     public static String errorMessage(Throwable e) {        
         return errorMessage(e, MessageFormat.format(UNEXPECTED_ERROR, e.getMessage()));
     }
+    
+    public static Boolean checkExceptionType(Exception e) {
```

Comment:
The method name doesn't provide much context to its functionality. What about naming it something like `isCorrelationKeyAlreadyExists`?

Reply:
done